package.json

```diff
@@ -22,15 +24,19 @@
     "url": "https://github.com/azeemba/eslint-plugin-json/issues"
   },
   "dependencies": {
-    "vscode-json-languageservice": "^3.2.1"
+    "vscode-json-languageservice": "^3.3.3"
   },
   "devDependencies": {
     "chai": "^4.2.0",
-    "eslint": "~5.1.0",
-    "mocha": "^5.2.0"
+    "eslint": "^5.1.0",
+    "eslint-config-prettier": "^6.2.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "lodash": "^4.17.15",
```

Comment:
Looks like lodash is not just a dev dependency right? So we should move it to be a main dependency?

Reply:
hell yeah! Good catch :)